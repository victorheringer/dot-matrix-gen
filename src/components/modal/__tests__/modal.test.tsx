import { render } from "@testing-library/react";
import { Modal } from "../index";

describe("Modal", () => {
  it("should render correctly a visible modal", () => {
    const tree = render(<Modal visible>Test</Modal>).asFragment();
    expect(tree).toMatchSnapshot();
  });

  it("should not render correctly a hidden modal", () => {
    const tree = render(<Modal visible={false}>Test</Modal>).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
