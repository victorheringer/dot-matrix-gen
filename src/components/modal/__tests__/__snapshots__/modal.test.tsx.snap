// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should not render correctly a hidden modal 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ dtlpkf"
  >
    Test
  </div>
</DocumentFragment>
`;

exports[`Modal should render correctly a visible modal 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ hqrsbG"
  >
    Test
  </div>
</DocumentFragment>
`;
